api/v1alpha1/bgppeer_webhook.go

```diff
@@ -89,6 +107,17 @@ func (bgpPeer *BGPPeer) validateBGPPeer(existingBGPPeersList *BGPPeerList) error
 	return err
 }
 
+func (bgpPeer *BGPPeer) validateBGPPeersKeepaliveTime(existingBGPPeersList *BGPPeerList) *field.Error {
+	ht := bgpPeer.Spec.HoldTime
+	ka := bgpPeer.Spec.KeepaliveTime
+
+	if ka > ht {
+		return field.Invalid(field.NewPath("spec").Child("KeepaliveTime"), ka,
```

Comment:
The `, ka` is not necessary I think

Reply:
u need the field to have all field.Error needs